lib/Elastica/QueryBuilder/DSL/Aggregation.php

```diff
@@ -194,10 +192,8 @@ public function cardinality($name)
      * geo bounds aggregation.
      *
      * @link http://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-geobounds-aggregation.html
-     *
-     * @param string $name
      */
-    public function geo_bounds($name)
+    public function geo_bounds()
```

Comment:
If this is implemented at some point in time `$name` would be required again (like in all aggregations) and produce a BC break.